humidity/lib/index.js

```diff
@@ -9,11 +9,18 @@ const Piloted = require('piloted');
 const Seneca = require('seneca');
 
 
+const internals = {};
+
+process.on('SIGHUP', () => {
```

Comment:
This is more for debugging purposes to see when SIGHUP was sent to the process, which will be when a backend changes.